src/signals/incident/definitions/wizard-step-3-contact.js

```diff
@@ -32,7 +30,7 @@ export default {
         },
         render: FormComponents.WithHeading,
         options: {
-          validators: [Validators.maxLength(17), validatePhoneNumber],
+          validators: [validatePhoneNumber, '17'],
```

Comment:
Voelt een beetje random om alleen een nummer toe te voegen als validator. Zie dat het voor de maxLength staat, maar wat als je minLength wil toevoegen? Misschien beter om dit gewoon maxLength(17) te maken oid.

Reply:
Ik ga deze in een aparte story finetunen denkik. Nu zaak dat het geheel door business getest kan worden. Kunnen we weer in fijne kleine stories dingen verbeteren:)